plugins/reporters/fossid/src/main/resources/templates/asciidoc/fossid_snippet.ftl

```diff
@@ -94,6 +94,46 @@ ${sourceLocation.startLine}-${sourceLocation.endLine}
 | ${snippet.purl!""}
 | ${snippet.licenses!""} | ${snippetFilePath} | ${snippet.provenance.sourceArtifact.url!""}[URL]
 | ${snippet.score!""} | ${snippet.additionalData["releaseDate"]}
+6+a|
```

Comment:
It this line intended? The syntax looks a bit odd.

Reply:
Yes

`6+` is to span 6 columns:
https://docs.asciidoctor.org/asciidoc/latest/tables/span-cells/

For the `a` I don't remember exactly. I think it was the `AsciiDoc` format that was required:
https://docs.asciidoctor.org/asciidoc/latest/tables/format-cell-content/